packages/sanity/src/core/form/inputs/files/ImageInput/ImageActionsMenu.tsx

```diff
@@ -82,34 +89,38 @@ export function ImageActionsMenu(props: ImageActionsMenuProps) {
 
   return (
     <MenuActionsWrapper data-buttons space={1} padding={2}>
-      {showEdit && (
-        <Button
-          aria-label="Open image edit dialog"
-          data-testid="options-menu-edit-details"
-          icon={CropIcon}
-          mode="ghost"
-          onClick={onEdit}
-          ref={setHotspotButtonElement}
-        />
-      )}
-      {/* Using a customized Popover instead of MenuButton because a MenuButton will close on click
+      <TooltipDelayGroupProvider delay={{open: 400}}>
```

Comment:
The delay used for these delay providers should be consistent across the studio – suggest that we avoid a magic number here and define a shared constant